packages/run-utils/tests/main.mjs

```diff
@@ -1,14 +1,14 @@
-'use strict'
+/* eslint-disable-next-line ava/no-ignored-test-files */
```

Comment:
This file must be `.mjs` due to `ava` not working well in monorepos mixing CommonJS and ESM packages. This also requires this `eslint-disable-next-line` comment. Once all packages of this monorepos are switched to ESM, this file can be renamed to `.js` and this comment can be removed.